src/stories/Quiz/Component/QuizResults/ResultCard.stories.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { expect } from '@storybook/jest';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { within } from '@storybook/testing-library';
+import ResultCard from '../../../../components/ResultCard/ResultCard';
+import QuizResultsDecorator from './QuizResultsDecorator';
+
+const product = {
+  value: 'This is a result card item',
+  data: {
+    id: '1',
+    price: 200,
+    salePrice: 150,
+    image_url: 'https://demo.constructor.io/sandbox_files/farmstandquizassets/HiThereNameInput.png',
+  },
+};
+
+const meta: Meta<typeof ResultCard> = {
+  title: 'Quiz/Results/ResultCard',
+  component: ResultCard,
+  argTypes: {},
+};
+
+export default meta;
+type Story = StoryObj<typeof ResultCard>;
+
+export const ResultCardWithRegularPrice: Story = {
```

Comment:
We might need something like a container here
![image](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/a72edac4-c284-4cef-973b-bf845f74db0f)